#88 Create Login Page using Login component

laila1222 opened the pull request:
- Created component of login form 
- Created login page - @rakhshanda-rashid

rakhshanda-rashid committed:
I made a button components for all the buttons in login form,i used font awsome and made a reuseable component out of it..

rakhshanda-rashid committed:
I made a button components for all the buttons in login form,i used font awsome and made a reuseable component out of it..

rakhshanda-rashid committed:
I made a button components for all the buttons in login form,i used font awsome and made a reuseable component out of it..

rakhshanda-rashid committed:
Login Form Email Address component with responsive feature

rakhshanda-rashid committed:
Login Form Email Address component with responsive feature

rakhshanda-rashid committed:
login form password field component with responsive feature

rakhshanda-rashid committed:
login form password field component with responsive feature

rakhshanda-rashid committed:
login form title

rakhshanda-rashid committed:
login form line

rakhshanda-rashid committed:
Merge branch 'master' of https://github.com/HackYourFuture-CPH/FP-annotatetheweb-class10 into buttons

rakhshanda-rashid committed:
All files formatted done under Button component

rakhshanda-rashid committed:
New component for login form field,its reusable and responsive

rakhshanda-rashid committed:
changes done

rakhshanda-rashid committed:
changes done

rakhshanda-rashid committed:
changes done

rakhshanda-rashid committed:
test on onclick props

rakhshanda-rashid committed:
file added

rakhshanda-rashid committed:
echo

rakhshanda-rashid committed:
changes done

rakhshanda-rashid committed:
changes done

rakhshanda-rashid committed:
changes

rakhshanda-rashid committed:
conflict fixed

rakhshanda-rashid committed:
changes

rakhshanda-rashid committed:
test file for button component

rakhshanda-rashid committed:
changes done in FormButton naming

rakhshanda-rashid committed:
changes

rakhshanda-rashid committed:
test file import FormButton spelling error fixed

rakhshanda-rashid committed:
aa

rakhshanda-rashid committed:
login page files created

rakhshanda-rashid committed:
form login

rakhshanda-rashid committed:
logiform linecomponent

rakhshanda-rashid committed:
header register

rakhshanda-rashid committed:
header register

rakhshanda-rashid committed:
form login

rakhshanda-rashid committed:
changes

rakhshanda-rashid committed:
login page is implemented

rakhshanda-rashid committed:
conflict fixed

rakhshanda-rashid committed:
css files for components changes bit according to reuirements

laila1222 committed:
added login page

rakhshanda-rashid committed:
css files settings fixed

rakhshanda-rashid committed:
build error fixed

rakhshanda-rashid committed:
build errors fixed

rakhshanda-rashid committed:
unnecessary files deleted

bytewiz reviewed (changes requested):
There is a lot of outdated changes here and files that should not be changed e.g. all files with `git` should not be here please remove them and we don't use enzyme for testing we use `react-testing-library` and the formatting is also a but off and please don't change the jest configurations...

bytewiz commented:
Please rename `FP-annotatetheweb-class10.git ` to `.git`

rakhshanda-rashid committed:
.git file deleted

laila1222 committed:
Deleted separate stories

We do not need a separate stories folder/file, because according to the storybook configs, all files that contain the word 'story' will be loaded as stories.

laila1222 committed:
Delete echo

Deleted empty echo - there were no changes listed.

laila1222 committed:
Delete package.json

laila1222 committed:
restored package.json from develop

laila1222 committed:
deleted empty file, deleted change in index.html

laila1222 committed:
restored app.test.js

laila1222 committed:
removed change from index.html

laila1222 committed:
restored .env.example file

laila1222 reviewed (commented):
I removed the files that shouldn't have been changed from this branch.

mo3ni commented on src/client/components/SignInImage/SignInImage.component.jsx:
```diff
@@ -0,0 +1,14 @@
+  )
+};
+
+export default Image;
```
do we need to create separate component for just an image or just define it inside sign in page!

laila1222 replied:
Done!

mo3ni commented on src/client/components/Header/Header.css:
```diff
@@ -1,27 +1,30 @@
-  top: 50px;
+  /* weight: 13rem;
+  left: -10px;
+  top: 15px; */
```
remove comments

mo3ni commented on src/client/components/Header/Header.css:
```diff
@@ -1,27 +1,30 @@
+  width: 14rem;
+  /* height: px;
+  left: -2500px;
+  top: -35px; */
```
remove comments

mo3ni commented on src/client/components/Header/Header.css:
```diff
@@ -1,27 +1,30 @@
+  font-family: Pacifico;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 1.5rem;
```
2 logo css property with different values

laila1222 committed:
removed commented lines

laila1222 committed:
deleted loginImage component, created <img> instead

bytewiz committed:
Merge branch 'develop' into 88/loginPage/19.12

bytewiz reviewed (approved):
Think that should it 👍
